submissions/semenjob/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <ul>
+        <li><a href="#">Mail</a></li>
+        <li><a href="#">Images</a></li>
+      </ul>
+      <input
+        tabindex="1"
+        class="popup__checkbox"
+        type="checkbox"
+        id="popup__checkbox"
+      />
+      <label for="popup__checkbox" class="popup__checkbox__label">
+        <img src="./icon/popup-button.png" alt="" />
+      </label>
+      <nav class="popup">
+        <ul class="popup__list">
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/apps.png" alt="" />
+            <span>Apps</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/bell.png" alt="" />
+            <span>Bell</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/blogs.png" alt="" />
+            <span>Blogs</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/calendar.png" alt="" />
+            <span>Calendar</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/chat.png" alt="" />
+            <span>Chat</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/classroom.png" alt="" />
+            <span>Classroom</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/collection.png" alt="" />
+            <span>Collection</span>
+          </li>
+          <li class="popup__item" tabindex="2">
+            <img class="popup__icon" src="./icon/contacts.png" alt="" />
+            <span>Contacts</span>
+          </li>
+          <li class="more">
+            <input
+              class="more__popup__checkbox"
+              type="checkbox"
+              id="more__popup__checkbox"
+              tabindex="3"
+            />
+            <label
+              class="more__popup__checkbox__label"
+              for="more__popup__checkbox"
+              >--More--</label
```

Comment:
Please separate responsibilities, HTML uses just for content and CSS for styles

Reply:
@kasionio What did you mean here? I didn't use styles in html

Reply:
Does >--  have any meaning? And, please, don't styling in a such way

Reply:
@kasionio I got it, thank you for explaining

Reply:
You are welcome